grow_cluster_test.py

```diff
@@ -60,10 +60,10 @@ def get_stress_cmd(self, mode='write', duration=None):
         :rtype: basestring
         """
         ip = self.db_cluster.get_node_private_ips()[0]
-        population_size = self.params.get('cassandra_stress_population_size', default=1000000)
+        population_size = self.params.get('cassandra_stress_population_size') or 1000000
```

Comment:
why do you keep defaults? it is set in `test-cases/features/scale-cluster.yaml`

Reply:
we have no default for that. If another yaml will be added to use this test and this parameter will be missed there, do we wnat this test will fail?

Reply:
that will be the default